src/Tests/Microsoft.NET.TestFramework/Commands/TestCommand.cs

```diff
@@ -83,7 +95,21 @@ public ProcessStartInfo GetProcessStartInfo(params string[] args)
         public CommandResult Execute(params string[] args)
         {
             IEnumerable<string> enumerableArgs = args;
-            return Execute(enumerableArgs);
+
+            if (RetryCommand)
+            {
+                return ExecuteWithRetry(
+                    action: () => Execute(enumerableArgs),
+                    isSuccess: (result) => result.ExitCode == 0,
+                    maxRetryCount: 2,
```

Comment:
Is there an overload that allows setting the max retry count to a different value?